### PR TITLE
[Cleanup] Extracting unused/unshared CSS

### PR DESCRIPTION
**Pull request by kdonovan**
## Description
As part of [this ticket](https://www.notion.so/teamshares/Polish-pass-UI-5b495cc24f914ddeb9f81438769e318c?pvs=4#2788f41765b04446b037f224220ecccf) I needed to override a style in OS that was being inherited from shared-ui (I have a mock where `.ts-table__header-row p` is _not_ bold).  While investigating whether I should modify the upstream or just override locally, I found it wasn't actually used in any other codebase.  And neighboring code for styling flash messages was [only used in buyout](https://github.com/teamshares/buyout-app/pull/1049) (which is using an older pattern/we don't _want_ to replicate that across codebases). 

_tl;dr: we can remove some stuff (either unused or only by a single codebase) and simplify our CSS a bit._

### CODE REVIEW

**Review by CrookedGrin — APPROVED** [review]
I'm all in favor of removing extra CSS, as long as we're sure none of it is being used in shared components or libraries.
